refs #7: Some simple guides for install and initial use

siglocpp opened the pull request:
Since I noticed the open Ticket #7 and just started using Rosalina, I want to contribute some documentation which would helped me to get started.

As I don't know, how you want to make guides accessible, I just wrote two small *.md files and added some pictures to your Documentation folder. 
Change or link to them as you see fit.

Commit:
refs #7

siglocpp committed:
removed dead img

Eastrall commented:
Hi, thanks for contributing to the project with some documentation. It a good start!

Maybe the "How to Install" doc should be on the main `README.md` file instead. Also, Rosalina has been added to OpenUPM: https://openupm.com/packages/com.eastylabs.rosalina/ thus, we can also install the package from there.

Commit:
refs #7: added OpenUPM installation, added some infos and links to readme

Commit:
Merge branch 'main' of github.com:siglocpp/Rosalina

siglocpp commented:
I added the OpenUPM option to the HowToInstall.md.
I also added a short description with a link to the detailed version for HowToInstall to the readme.
Additionaly added a link to the HowToCreateFiles to the readme.
